cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -365,6 +365,60 @@ def get_all(cls, filters=None, credentials=None):
 
         return cls._entity_repo.get_all(filters, credentials)
 
+    @check_do_enabled
```

Comment:
A minor question. Why moving this to the `classmethod` section?

Reply:
Because it was in the download one too. Not sure what is your doubt

Reply:
Usually, we keep the structure `classmethod`s, `staticmethod`s, `instance methods`, and now it's mixed, but it's a minor thing, just asking if there was a reason.